### PR TITLE
Remove memory leak from CustomImageClassifier.java

### PR DESCRIPTION
**Pull request by FuadRafid**
The imgData variable in convertBitmapToByteBuffer function was being assigned a new byte[] each time, and the last used byte[] were not being removed from memory which was causing a memory leak. The variable was made a property of the class and was initiated only once, cleared and reused again.

### CODE REVIEW

File: mlkit/app/src/main/java/com/google/firebase/samples/apps/mlkit/java/custommodel/CustomImageClassifier.java [hunk 1]
```diff
@@ -261,16 +264,18 @@ public void onFailure(@NonNull Exception e) {
...
-    private synchronized ByteBuffer convertBitmapToByteBuffer(
+    private synchronized void convertBitmapToByteBuffer(
             ByteBuffer buffer, int width, int height) {
         int bytesPerChannel = mUseQuantizedModel ? QUANT_NUM_OF_BYTES_PER_CHANNEL :
                 FLOAT_NUM_OF_BYTES_PER_CHANNEL;
-        ByteBuffer imgData =
-                ByteBuffer.allocateDirect(
-                        bytesPerChannel * DIM_BATCH_SIZE * DIM_IMG_SIZE_X * DIM_IMG_SIZE_Y * DIM_PIXEL_SIZE);
-        imgData.order(ByteOrder.nativeOrder());
+        if (this.imgData == null)
```
Comment (samtstern) [inline mlkit/app/src/main/java/com/google/firebase/samples/apps/mlkit/java/custommodel/CustomImageClassifier.java]: Could you add `{` and `}`  on all `if/else` statements to be consistent with our Java style?

**Comment by samtstern** [comment]
Approved with one small comment, thanks for this!

File: mlkit/app/src/main/java/com/google/firebase/samples/apps/mlkit/java/custommodel/CustomImageClassifier.java (hunk 1 above)
Comment (FuadRafid) [reply mlkit/app/src/main/java/com/google/firebase/samples/apps/mlkit/java/custommodel/CustomImageClassifier.java]: Alright doing it. Thanks for the feedback.

**Comment by FuadRafid** [comment]
@samtstern made the changes. Please review when convenient. Thank you.

**Comment by samtstern** [comment]
@FuadRafid thanks!  Looks great.

**Pull request merged**